### DISCUSSION TITLE
A sudden new problem with my GPU

### DISCUSSION THREAD

**Discussion by 7HRAMZI** [lllyasviel/Fooocus#2797] [Q&A] [answered]
Hello brothers and sisters
Hi, I have a problem with my GPU where it suddenly changed and became slower and less efficient what is the reason
For example, instead of taking 15 minutes or less, it now takes 40, 45 minutes or more. In addition, the only consumption in the GPU is V-RAM, with a noticeable increase in temperature and a decrease in performance and productivity.
Knowing that this is the first time this problem has happened to me
![sqfdqs](https://github.com/lllyasviel/Fooocus/assets/159185564/5bd32a70-3e36-4b3d-8a7e-5a61eefed154)
Please help me...

**Comment by mashb1t** [comment, marked as answer]
Your GPU does not neet the minimum requirements, see https://github.com/lllyasviel/Fooocus?tab=readme-ov-file#minimal-requirement. Nevertheless, if it worked before, some questions to you:
- since when does this behavior occur?
- did you do a driver update or other modifications around the time of the change?
- have your interactions with Fooocus changed, e.g. using other features?

**Reply by 7HRAMZI** [reply]
HI, thanks for the response and this is my answer And I'm really sorry I'm late

23April 2024 at 9:45 PM
0.I started using it last March and it worked perfectly
1. I encountered this issue around April 22, 2024. This is the first time. Previously it was working fine without problems.
2. Graphics card update number is 552.22, nvidia studio driver version.
3. When the problem appeared, I deleted the program and downloaded it again, and the same problem still exists.
4. I haven't changed my style when using Fooocus
Update

April 25, 2024 at 12:20 PM
Update: I've fixed the issue
First I deleted the program
Second, windows 10 Factory settings "Rest This PC" Delete everything in the system partition
Third, I defragmented the hard drive
Fourth, re-download the program
Thus, it returned to its previous state and worked properly
Note: I didn't know this would solve the problem...

April 27, 2024 at 4:05 PM
The date the response was sent

**Reply by 7HRAMZI** [reply]
Soon I will do several experiments with this program to find out the cause of the problem and I will inform you about it
Sorry in advance for being late
